include/onnxruntime/core/session/onnxruntime_cxx_api.h

```diff
@@ -44,8 +44,15 @@ struct Global {
   static const OrtApi& api_;
 };
 
+#ifdef EXCLUDE_REFERENCE_TO_ORT_DLL
+OrtApi stub_api;
```

Comment:
Please don't define a variable in a C++ header file.

Reply:
@snnn It's a temporary workaround until I can refactor the C++ headers so that custom op stuff is separate from the C++ header that takes a dependency on the DLL.

Reply:
Ok.

Reply:
I'm worrying, the "Global<T>::api_" object may get destructed twice, like a double free.

Reply:
We can discuss it later.